index.html

```diff
@@ -16,10 +17,26 @@
 
 <body>
 
-  
-     
   <div class="container home">
+
+      <nav>
+          <h1>Fun Bus</h1>
+          <a href="#">Home</a>
+          <a href="#">About Us</a>
+          <a href="#">Blog</a>
+          <a href="#">Contact</a>
+     </nav>
    
+     <img class = "bus" src="/img/fun-bus.jpg" alt="bus">
```

Comment:
I think the reason the image isn't showing might be because the source is '/img/fun-bus.jpg' and not 'img/fun-bus.jpg'